### PR TITLE
Leaf/mint napkin/new update notification

### LINKED ISSUE #102: [SECONDARY FEATURE] New Updates Notification
### User Story

As a student, I want to know if the extension gets any updates that is not intrusive on my overall experience.

### Technical Tasks

- Create a notification using MUI's [Alert component](https://mui.com/material-ui/react-alert/). The notification should continue to appear until the user closes the notification in the extension window.
- The notification should appear when there is a new version of the extension.

### Acceptance Criteria

- When the extension receives an update, a notification appears in the extension window page.

### PR DESCRIPTION
**Pull request by MinT-Napkin**
New Update Notification Component (Closes #102)

Alert should appear right above the search bar in the extension window

Description:
- When there is an update available, the alert appears
- If closed by the user, it is locally set to never appear again until the extension is updated so that the next update will have the alert

Testing:
- **onUpdateAvailable and onInstalled chrome runtime methods I do not think have a local way to test them as far as I know**
- To view / see the alert functionality, set the alertVisibity state to true, it should appear in the extension window
- If you close it and it doesn't appear again, its working as intended but if you'd like to reset that you can copy and paste this piece of code in the console window when you inspect pop-up when you right click the extension. It should appear again when you reopen the extension window (assuming you have alertVisibility set to true in the code).

chrome.storage.local.set({ alertClosed: false }, () => {
  if (chrome.runtime.lastError) {
    console.error(chrome.runtime.lastError);
  }
});

### CODE REVIEW

File: src/components/UpdateAlert.tsx
Comment (GuyWhoCode) [inline src/components/UpdateAlert.tsx:4]: ESLint should have technically flagged this function for not having JSDocs, so can you add some? Look at some of the other component folders for examples of JSDocs (comments above each function)

**Review by GuyWhoCode — APPROVED** [review]
Overall looks good to me. I would wait until @PattyCakesxD reviews it and publishes a test version of this on the Chrome web store before merging.

File: src/App.tsx
Comment (MinT-Napkin) [reply src/App.tsx:23]: Aight, thanks for letting me know on that.

File: src/components/UpdateAlert.tsx
Comment (MinT-Napkin) [reply src/components/UpdateAlert.tsx:4]: Yeah I can do that.

**Review by PattyCakesxD — APPROVED** [review]
lgtm

**Pull request merged**
